joint_trajectory_controller/test/config/test_joint_trajectory_controller.yaml

```diff
@@ -10,3 +10,33 @@ test_joint_trajectory_controller:
   state_interfaces:
     - position
     - velocity
+
+  command_joints:
```

Comment:
Shouldn't throw this an error, if the size of `command_joints` is different than `joints`? IMHO controller chaining is not implemented for JTC yet, you might have a look if this parameter is used and what is expected there.